JSZVCR/Classes/Recorder/JSZVCRRecorder.m

```diff
@@ -84,6 +85,7 @@ - (void)recordTask:(NSURLSessionTask *)task didReceiveResponse:(NSURLResponse *)
             NSLog(@"already had response: %@", recording.response);
         }
         recording.response = [JSZVCRResponse responseWithResponse:response];
+        recording.data = nil;
```

Comment:
This appears to be needed because of reusing a recording during some of the tests: if we don't `nil` it here, we ended up with *two* copies of the same data. I'm not 100% sure why this reuse is happening.